src/resources/model.js

```diff
@@ -62,6 +62,7 @@ pc.extend(pc, function () {
             var data = asset.data;
 
             resource.meshInstances.forEach(function (meshInstance, i) {
+                meshInstance.mesh.area = data.area;

```

Comment:
Hm OK I thought that the area is calculated by the engine? If not then maybe there is no need to duplicate this in the mesh?